README.md

```diff
@@ -26,10 +26,20 @@ The GOV.UK Prototype Kit is maintained by the Government Digital Service. If you
 * email [govuk-design-system-support@digital.cabinet-office.gov.uk](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk) 
 * [get in touch on Slack](https://ukgovernmentdigital.slack.com/messages/prototype-kit)([open in app](slack://channel?team=T04V6EBTR&amp;id=C0647LW4R)) 
 * [view known issues on GitHub](https://github.com/alphagov/govuk-prototype-kit/issues)
-        
+
 ## Contributing
 
 If you’ve got an idea or suggestion you can:
 
 * [get in touch on the developer Slack channel](https://ukgovernmentdigital.slack.com/messages/prototype-kit-dev)([open in app](slack://channel?team=T04V6EBTR&amp;id=C0E1063DW))
 * [create a GitHub issue](https://github.com/alphagov/govuk-prototype-kit/issues)
+
+## Deploying to GOV.UK PaaS
+
+If you want to deploy your prototype to [GOV.UK PaaS](https://www.cloud.service.gov.uk) you can use the existing [application manifest](manifest.yml) and push using the following command:
+
+```
+cf push --var username=myusername --var password=secret --var prototype-name=my-new-service
+```
+
```

Comment:
@tlwr @richardTowers I'm compiling your collective wisdom on setting up genuinely secret passwords in this suggestion:

```suggestion
If your prototype repo is public but you need the password to be secret, you can use environment variables when pushing your app:

```
cf push --var username=hello --var "password=$PASSWORD" --var prototype-name=my-new-service
```
If you've set up continuous deployment for your prototype, check the recommended ways to handle secrets. For example, [how Travis CI works with encrypted variables](https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml) or [how to create and use secrets with GitHub Actions](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables).
```

Reply:
I've attempted to incorporate these suggestions into the tutorial page.